api/conf/settings/spectacular.py

```diff
@@ -41,10 +42,11 @@
     "SORT_OPERATION_PARAMETERS": False,
     "EXTENSIONS_INFO": {
         "x-logo": {
-            "url": "https://raw.githubusercontent.com/WordPress/openverse/HEAD/brand/logo.svg",
+            "url": "https://raw.githubusercontent.com/WordPress/openverse/HEAD/documentation/meta/brand/logo.svg",
             "backgroundColor": "#fafafa",
         }
     },
+    "SWAGGER_UI_FAVICON_HREF": f"{STATIC_URL}favicon.ico",
```

Comment:
Setting the URL from the root makes it work for swagger: http://localhost:50280/v1/swagger/#/ 
```suggestion
    "SWAGGER_UI_FAVICON_HREF": f"/{STATIC_URL}favicon.ico",
```
Although this does not solve the problem for the index.

Reply:
Ah yes, I think its worth adding the leading slash to `STATIC_URL` itself in the conf, but that can be addressed separately.

https://github.com/WordPress/openverse/blob/18a6606cd5ef56d74d9b3f592e8c899148474aee/api/conf/settings/static.py#L10